recipes/_configure_inputs.rb

```diff
@@ -11,15 +11,14 @@
 input_stanzas = CernerSplunk::LWRP.convert_monitors node['splunk']['monitors'], node['splunk']['main_project_index'], base_hash
 
 if %i[server cluster_slave].include? node['splunk']['node_type']
-  bag = CernerSplunk.my_cluster_data(node)
+  cluster, bag = CernerSplunk.my_cluster(node)
   port = bag['receiver_settings']
   port = port['splunktcp'] if port
   port = port['port'] if port
-
   if port
-    input_stanzas["splunktcp://:#{port}"] = {}
-    input_stanzas["splunktcp://:#{port}"]['disabled'] = 0
-    input_stanzas["splunktcp://:#{port}"]['connection_host'] = 'none'
+    input_stanzas["splunktcp://#{port}"] = {}
```

Comment:
I ran into this issue https://answers.splunk.com/answers/341027/why-am-i-unable-to-connect-my-forwarder-to-an-inde.html and the fix suggested by MuS fixed it.